### PR TITLE
Add a few precompiles

### PR DESCRIPTION
**Pull request by timholy**

### CODE REVIEW

File: src/FFTW.jl [hunk 1]
```diff
@@ -91,4 +91,9 @@ end
 include("fft.jl")
 include("dct.jl")
 
+if Base.VERSION >= v"1.4.2"
```
Comment (ararslan) [inline src/FFTW.jl:94]: Can you add a comment about why it's this particular version?

Comment (timholy) [inline src/FFTW.jl:94]:
```suggestion
if Base.VERSION >= v"1.4.2"     # avoid potential segfaults, see https://github.com/JuliaLang/julia/pull/35378
```

Comment (timholy) [reply src/FFTW.jl]: Done

**Review by ararslan — APPROVED** [review]

**Comment by timholy** [comment]
Thanks for the review, @ararslan!

**Comment by ararslan** [comment]
My pleasure :)

**Pull request merged**